src/commands/ssh-resolve.ts

```diff
@@ -116,8 +116,11 @@ const sshResolveAction = async (
 
   const p0Executable = bootstrapConfig.appPath;
 
+  // Replace any characters that don't make a good 
+  const sanitizedDestination = destination.replace(/[^a-zA-Z0-9-_]/g, "_");
+
   const data = `Host ${destination}
-  Hostname ${destination}
+  Hostname ${sanitizedDestination}
```

Comment:
The Hostname has to be mapped to the sanitized version to avoid the following error:
```
Destination cannot contain a forward slash (/).
```